core/src/main/java/macrobase/analysis/summary/itemset/RiskRatio.java

```diff
@@ -0,0 +1,62 @@
+package macrobase.analysis.summary.itemset;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class RiskRatio {
+    private static final Logger log = LoggerFactory.getLogger(RiskRatio.class);
+
+    private static double computeDouble(double exposedInlierCount,
+                                        double exposedOutlierCount,
+                                        double totalInliers,
+                                        double totalOutliers) {
+        double totalExposedCount = exposedInlierCount + exposedOutlierCount;
+        double unexposedOutlierCount = (totalOutliers - exposedOutlierCount);
+        double totalMinusExposedCount = totalInliers + totalOutliers - totalExposedCount;
+
+        // no exposure occurred
+        if (totalExposedCount == 0) {
+            log.error("Computing risk ratio with no exposure.");
+            return 0;
+        }
+
+        // we only exposed this ratio, everything matched!
+        if (totalMinusExposedCount == 0) {
+            return 0;
```

Comment:
It's not clear to me why when everything is exposed, the risk ratio should be 0.